pyppeteer/page.py

```diff
@@ -454,20 +479,13 @@ def _onCertificateError(self, event: Any) -> None:
 
         :arg str expression: XPath string to be evaluated.
         """
-        frame = self.mainFrame
-        if not frame:
-            raise PageError('no main frame.')
-        return await frame.xpath(expression)
+        return await self.mainFrame.xpath(expression)
 
-    #: alias to :meth:`querySelector`
+    # Shortcut aliases
     J = querySelector
-    #: alias to :meth:`querySelectorEval`
     Jeval = querySelectorEval
-    #: alias to :meth:`querySelectorAll`
     JJ = querySelectorAll
-    #: alias to :meth:`querySelectorAllEval`
     JJeval = querySelectorAllEval
-    #: alias to :meth:`xpath`
     Jx = xpath
 
     async def cookies(self, *urls: str) -> List[Dict[str, Union[str, int, bool]]]:
```

Comment:
shouldn't we be look for a sequence or urls in this case?
```suggestion
    async def cookies(self, *urls: Sequence[str]) -> List[Dict[str, Union[str, int, bool]]]:
```